[ADD] VoIP: Sales Calls Workflow (#12818)
odoo/documentation: content/applications/productivity/voip/sales_calls.rst

======================
 Sales calls with VoIP
======================

.. |VOIP| replace:: :abbr:`VoIP (Voice over Internet Protocol)`

Using |VOIP| increases call efficiency, reduces costs, and allows a salesperson to access the Odoo
database while on a call. Instead of relying on a separate phone for sales calls, a salesperson can
make calls directly from the Odoo database.

This article goes over setting up Odoo |VOIP| for a sales team that already has |VOIP| configured.

> If |VOIP| is **not** configured, refer to the  :doc:`set up VoIP <../voip>` documentation to set up
> |VOIP|.

Reviewer: I recommend using a :ref: to the specific section here, you'll need to add a custom anchor above the corresponding heading :)
```suggestion
If |VOIP| is **not** configured, refer to the  :ref:`configure VoIP <voip/configure>` documentation
to get started.
```

> .. example::

Reviewer: We typically [use seealso admonitions for linking to eLearning courses](https://github.com/odoo/documentation/pull/12818#discussion_r2038385158) :)
```suggestion
.. seealso::
```

actions, like sending an email to the customer, or pulling up their profile. Learn more about
:doc:`the documents a salesperson can access <voip_widget>` during a call.

The salesperson can also take some actions during the call:

- :icon:`fa-arrows-h` :guilabel:`(transfer)`: Transfer a call to another team member.
- :icon:`fa-microphone` :guilabel:`(microphone)`: The salesperson can mute themselves during a call.
- :icon:`fa-pause` :guilabel:`(pause)`: Place the caller on hold.

.. image:: sales_calls/voip-widget-call.png
   :alt: The VoIP widget opened to a call, showing the shortcut icons a salesperson has access to.

Handle call follow-ups
----------------------

The |VOIP| widget can handle some common follow-up tasks, but salespeople can still navigate Odoo
for tasks not available in the widget.

Here are some of the common follow-up tasks that can be managed from the |VOIP| widget:

- Send a follow-up email sharing new product offerings to the customer using the :icon:`fa-envelope`
  :guilabel:`(envelope)` icon.
- Update the customer's mailing address by clicking on the icon:`fa-user` :guilabel:`(user)` icon,
  which opens their profile.

> - Schedule a second call with the customer by clicking the :icon:`fa-clock` :guilabel:`(clock)`

Reviewer: Incorrect icon class here :)
```suggestion
- Schedule a second call with the customer by clicking the :icon:`fa-clock-o` :guilabel:`(clock)`
```
